R/tidyverse.R

```diff
@@ -32,6 +32,10 @@
 #' * `use_tidy_community()`: wrapper function around contributing, issues,
 #'   support, and code of conduct functions.
 #'
+#' * `use_tidy_github()`: creates standard tidyverse: issue template,
+#'   support resources document, contributing guidelines, and code of conduct in
+#'   an `.github/` subdirectory.
+#'
```

Comment:
creates standard tidyverse ?missing word?: issue template, ...

Reply:
```
creates standard tidyverse (then the colon was for the beginning of a list) issue template, support resources document…
```
I could easily just kill that colon though. It _does_ look super confusing in monospace.

I swapped "a" and "an" like three or four times (including changing the order to `a subdirectory, .github/` to avoid the issue entirely)…I'll put it back to "a."